lib/diplomat/kv.rb

```diff
@@ -42,6 +44,8 @@ def get key, options=nil, not_found=:reject, found=:return
       url += use_consistency(@options)
       url += dc(@options)
 
+      return_nil_values = (@options and @options[:nil_values])
```

Comment:
(Sorry for the slow reply, I was on vacation and away from computers.)

No, no need to change. After looking more closely plenty of other 'and's in the code. :-) I was just a little surprised to see then and.